[CS2103T-T11-2] Dukemon

sreesubbash opened the pull request:
@kohyida1997 @chrischenhui @dragontho @jascxx

ambhinav commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,17 +294,58 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* * *` |user |see the content of the word bank |study beforehand/make changes
+
+|`* * *` |young student |trivia questions to be gamified |enjoy the process
```
Perhaps you should write a more specific benefit instead of "enjoy the process"

ambhinav commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,17 +294,58 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* * *` |user |delete word banks |free up some memory when I don’t need it anymore
+
+|`* * *` |user |see the content of the word bank |study beforehand/make changes
```
May need to rephrase this use case. Can't really understand how the benefit relates to the "I want to..."

ambhinav commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,17 +294,58 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* *` |studious student |set and complete goals |have something to work towards
+
+|`* *` |student |see my test statistics |track my progress/improvement
```
Nice use case!

ambhinav commented on docs/DeveloperGuide.adoc:
```diff
@@ -347,8 +387,8 @@ _{More to be added}_
 .  Should work on any <<mainstream-os,mainstream OS>> as long as it has Java `11` or above installed.
-.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
 .  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+.  Users can export and import their <<word-bank,word banks>> or statistics.
```
This is a functional requirement

ambhinav commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,17 +294,58 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* *` |student |share and compare my results with my classmates |know where I stand
+
+|`* *` |student |partition the trivia |attempt questions that I'm comfortable with
```
Consider rephrasing "partition the trivia" to "sort according to difficulty" etc..

ambhinav reviewed (commented):
Good job, your project seems very interesting! 
Please look at the comments my team made regarding user stories and non-functional requirements

elsakoh commented on docs/DeveloperGuide.adoc:
```diff
@@ -44,14 +44,24 @@ The following class plays an important role at the architecture level:
+Holds the data of live game sessions in-memory.
+* <<Design-Storage,*`Storage`*>>: +
+Reads data from, and writes data to, the local hard disk.
+
```
Good introduction to your main components, easy to understand.

elsakoh commented on docs/DeveloperGuide.adoc:
```diff
@@ -12,7 +12,7 @@ ifdef::env-github[]
 :warning-caption: :warning:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level3/tree/master
+:repoURL: https://github.com/AY1920S1-CS2103T-T11-2/main
```
can change the header title!

ArunBeCoding commented on docs/DeveloperGuide.adoc:
```diff
@@ -131,6 +150,20 @@ As a more OOP model, we can store a `Tag` list in `Address Book`, which `Person`
 
+[[Design-Game]]
+=== Game component
+
```
Very good, very impressive

elsakoh commented on docs/DeveloperGuide.adoc:
```diff
@@ -154,30 +187,33 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 image::UndoRedoState1.png[]
 
-Step 3. The user executes `add n/David ...` to add a new person. The `add` command also calls `Model#commitAddressBook()`, causing another modified address book state to be saved into the `addressBookStateList`.
+Step 3. The user executes `stop` to abruptly end the game. The `add` command also calls `Model#commitAddressBook()`, causing another modified address book state to be saved into the `addressBookStateList`.
```
very clear explanation, well done!

WilliamRyank commented on docs/diagrams/GameClassDiagram.puml:
```diff
@@ -0,0 +1,40 @@
+Class HiddenOutside #FFFFFF
+HiddenOutside ..> Game
+HiddenOutside ..> Guess
+Game -- ReadOnlyWordBank :.clone() Cards from >
```
i do not think the `.clone()` should be in the UML diagram

C-likethis123 commented on docs/DeveloperGuide.adoc:
```diff
@@ -154,30 +187,33 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
-Step 2. The user executes `delete 5` command to delete the 5th person in the address book. The `delete` command calls `Model#commitAddressBook()`, causing the modified state of the address book after the `delete 5` command executes to be saved in the `addressBookStateList`, and the `currentStatePointer` is shifted to the newly inserted address book state.
+Step 2. The user enters `start difficult` command to begin the game. The `LogicManager` passes string to `Model#commitAddressBook()`, causing the modified state of the address book after the `delete 5` command executes to be saved in the `addressBookStateList`, and the `currentStatePointer` is shifted to the newly inserted address book state.
 
 image::UndoRedoState1.png[]
```
The diagrams do not fit the descriptions.

C-likethis123 commented on docs/DeveloperGuide.adoc:
```diff
@@ -131,6 +150,20 @@ As a more OOP model, we can store a `Tag` list in `Address Book`, which `Person`
+
+
+.Structure of the Game Component
+image::GameClassDiagram.png[]
```
A method should not be indicated in the association label
<img width="570" alt="Screenshot 2019-10-24 at 1 50 57 PM" src="https://user-images.githubusercontent.com/27060502/67457133-62ba0a00-f665-11e9-83bb-a222a2a9d916.png">

dragontho committed:
Merge pull request #193 from AY1920S1-CS2103T-T11-2/paul-8-ppp

added more pictures and cleaned up the bullet pointing

kohyida1997 committed:
Merge pull request #192 from AY1920S1-CS2103T-T11-2/yida-4-UGDG

Updated UG - Command Summary

chrischenhui committed:
Improve find command and update UG

chrischenhui committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into chenhui-2-codeQuality

# Conflicts:
#	docs/UserGuide.adoc

chrischenhui committed:
Fix some check style

chrischenhui committed:
Merge pull request #194 from AY1920S1-CS2103T-T11-2/chenhui-2-codeQuality

Chenhui 2 code quality

kohyida1997 committed:
Yida PPP additional edits

kohyida1997 committed:
Merging to stay updated with master

chrischenhui committed:
Implement more sample data.
Upon start, sample data will be automatically created, but
will not automatically create itself again after it has been removed by user.

chrischenhui committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into chenhui-2-codeQuality

chrischenhui committed:
Pass check style

chrischenhui committed:
Pass Checkstyle

sreesubbash committed:
Remove Exit from Game Mode

chrischenhui committed:
Pass checkstyle

sreesubbash committed:
Remove Exit from Game Mode

Remove Exit from Game Mode

chrischenhui committed:
Merge pull request #195 from AY1920S1-CS2103T-T11-2/chenhui-2-codeQuality

Chenhui 2 code quality

sreesubbash committed:
Switch Commands purpose made clearer

sreesubbash committed:
Switch Commands purpose made clearer

Switch Commands purpose made clearer

kohyida1997 committed:
Cleaning up GameCommand tests

kohyida1997 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into yida-4-UGDG

kohyida1997 committed:
Added AppManager diagram to DG

kohyida1997 committed:
Fix EOF error

kohyida1997 committed:
Updated Timer diagrams

kohyida1997 committed:
Updated DG Diagrams

kohyida1997 committed:
Fixing code quality

kohyida1997 committed:
Housekeeping of UG & DG, fixed PE Hints bugs

Housekeeping of UG & DG, fixed PE Hints bugs

chrischenhui committed:
Implement auto creation of revision wordbank, whenever a card is wrong in a game session. When card is correct, card get removed from the bank automatically.

chrischenhui committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into chenhui-8-revisionWordBank

chrischenhui committed:
Add javadoc and fix failing test

chrischenhui committed:
Merge pull request #199 from AY1920S1-CS2103T-T11-2/chenhui-8-revisionWordBank

Chenhui 8 revision word bank

jascxx committed:
Reorder dg

jascxx committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into jason-2-ppp

jascxx committed:
Fix

jascxx committed:
Merge pull request #200 from AY1920S1-CS2103T-T11-2/jason-2-ppp

Reorder DG

jascxx committed:
Update statistics class diagram

jascxx committed:
Merge pull request #201 from AY1920S1-CS2103T-T11-2/jason-2-ppp

Docs update

chrischenhui committed:
Fix revision bank bugs

kohyida1997 committed:
Subbash edits on my com

kohyida1997 committed:
Subbash edits on my com

chrischenhui committed:
Shift some code

chrischenhui committed:
Pass checkstyle

chrischenhui committed:
Fix build

chrischenhui committed:
Fix isValidMeaning check for revision meaning

chrischenhui committed:
Implement revision bank

chrischenhui committed:
Merge pull request #202 from AY1920S1-CS2103T-T11-2/chenhui-8-revisionWordBank

Chenhui 8 revision word bank

kohyida1997 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into yida-4-UGDG

sreesubbash committed:
Improved error messages in Parser Manager

sreesubbash committed:
Subbash 26 improve parse exception feedback

Subbash 26 improve parse exception feedback

kohyida1997 committed:
Revision to PPP

kohyida1997 committed:
Resolved conflicts with master

chrischenhui committed:
Pass addcommandtest and addcommandintegrationtest

kohyida1997 committed:
Refined tests

kohyida1997 committed:
Added activity diagram for Timer (with Hints)

kohyida1997 committed:
Merge pull request #204 from AY1920S1-CS2103T-T11-2/yida-5-MoreDocs

Edited PPP and DG, cleaned up tests for Card and GameTimer

chrischenhui committed:
Pass deletecommandtest

chrischenhui committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into chenhui-5-implementTests

# Conflicts:
#	src/main/java/seedu/address/model/ModelManager.java

chrischenhui committed:
Implement some importcommand test

chrischenhui committed:
Fix checkstyle

chrischenhui committed:
Merge pull request #205 from AY1920S1-CS2103T-T11-2/chenhui-5-implementTests

Chenhui 5 implement tests

chrischenhui committed:
Update ppp

chrischenhui committed:
Update ppp

kohyida1997 committed:
Updated Timer part for DG

kohyida1997 committed:
DG Changes

kohyida1997 committed:
Merge pull request #206 from AY1920S1-CS2103T-T11-2/yida-5-MoreDocs

Updating DG Timer and PPP

jascxx committed:
Fix wordbankstats not removed when wb is removed

jascxx committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into jason-2-ppp

jascxx committed:
Fix style

jascxx committed:
Merge pull request #207 from AY1920S1-CS2103T-T11-2/jason-2-ppp

Fix wordbankstats not removed when wb is removed, and most played wordbank showing an unplayed wordbank

chrischenhui committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into chenhui-3-UGDG

chrischenhui committed:
Update DG PPP

chrischenhui committed:
Update diagrams in DG

chrischenhui committed:
Update DG

jascxx committed:
Update DG

jascxx committed:
Update DG

jascxx committed:
Merge pull request #209 from AY1920S1-CS2103T-T11-2/jason-2-ppp

Update DG

jascxx committed:
Add note to UG

jascxx committed:
Implement reset command for wordbankstatistics

jascxx committed:
Implement reset command

jascxx committed:
Fix style

jascxx committed:
Merge pull request #210 from AY1920S1-CS2103T-T11-2/jason-2-ppp

Implement reset command

therealdragontho committed:
Removed redundant variables in ModularDisplay

therealdragontho committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into paul-8-ppp

therealdragontho committed:
Changed UI part of DG to fit Dukemon

therealdragontho committed:
Added new parts to my PPP and cleaned up some code

dragontho committed:
Merge pull request #211 from AY1920S1-CS2103T-T11-2/paul-8-ppp

Edited UI part of DG, cleaned up code and edited PPP

chrischenhui committed:
Complete DG

chrischenhui committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into chenhui-3-UGDG

# Conflicts:
#	src/main/java/seedu/address/storage/statistics/JsonWordBankStatisticsListStorage.java

chrischenhui committed:
Fix compile error n checkstyle

chrischenhui committed:
Merge pull request #212 from AY1920S1-CS2103T-T11-2/chenhui-3-UGDG

Chenhui 3 ugdg

kohyida1997 committed:
Formatting PPP

chrischenhui committed:
Polish DG

sreesubbash committed:
FIX comments

sreesubbash committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main

sreesubbash committed:
Move some tests to ParserManager

jascxx committed:
Fix UG mistake

jascxx committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into jason-2-ppp

jascxx committed:
Reorder UG

jascxx committed:
Fix font

jascxx committed:
Fix grammar

jascxx committed:
Minor update

jascxx committed:
Merge pull request #213 from AY1920S1-CS2103T-T11-2/jason-2-ppp

Fix UG

jascxx committed:
Minor UG update

jascxx committed:
Minor format update

jascxx committed:
Merge pull request #214 from AY1920S1-CS2103T-T11-2/jason-2-ppp

Minor UG update

jascxx committed:
Very minor UG update

jascxx committed:
Merge pull request #215 from AY1920S1-CS2103T-T11-2/jason-2-ppp

Very minor UG update

chrischenhui committed:
Change pronoun "I" to "we"

chrischenhui committed:
Improve some code quality

chrischenhui committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into chenhui-3-UGDG

kohyida1997 committed:
PPP Update

kohyida1997 committed:
Resolved Merge conflicts

kohyida1997 committed:
Merge pull request #217 from AY1920S1-CS2103T-T11-2/yida-5-MoreDocs

PPP Edits and Formatting

kohyida1997 committed:
Housekeeping, improving readability of code chunks

kohyida1997 committed:
Merge pull request #218 from AY1920S1-CS2103T-T11-2/yida-5-MoreDocs

Housekeeping, improving readability of code chunks

chrischenhui committed:
Update PPP

chrischenhui committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into chenhui-3-UGDG

chrischenhui committed:
Update to DG, improve PPP

chrischenhui committed:
Merge pull request #219 from AY1920S1-CS2103T-T11-2/chenhui-3-UGDG

Chenhui 3 ugdg

kohyida1997 committed:
Updating DG for Hints

kohyida1997 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into yida-5-MoreDocs

chrischenhui committed:
Update PPP

sreesubbash committed:
Updated images

chrischenhui committed:
Update PPP

sreesubbash committed:
Final ppp update without latest diagrams

therealdragontho committed:
Cleaned up PPP

chrischenhui committed:
Polish PPP

kohyida1997 committed:
Optimization

chrischenhui committed:
Update ppp

kohyida1997 committed:
Optimization

chrischenhui committed:
Add author tags for classes I implemented. Common classes untouched

chrischenhui committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into chenhui-3-UGDG

kohyida1997 committed:
Cleaning up tests

kohyida1997 committed:
Minor changes

chrischenhui committed:
Improve some code quality

chrischenhui committed:
Update UG, DG, PPP

chrischenhui committed:
Update PPP, DG

chrischenhui committed:
Fix checkstyle

chrischenhui committed:
Fix some gradlew issues

chrischenhui committed:
Merge pull request #220 from AY1920S1-CS2103T-T11-2/chenhui-3-UGDG

Chenhui 3 ugdg

sreesubbash committed:
Final PPP

sreesubbash committed:
remove trailing spaces

sreesubbash committed:
Merge pull request #216 from AY1920S1-CS2103T-T11-2/subbash-27-finalPPPUpdate

Last update

sreesubbash committed:
Reset files to old version from subbash-26 branch

sreesubbash committed:
Reset files to old version from subbash-26 branch

Reset files to old version from subbash-26 branch

jascxx committed:
PPP

jascxx committed:
Minor update

jascxx committed:
Very minor update

kohyida1997 committed:
Changing Site Headers

kohyida1997 committed:
Resolved merge conflicts

kohyida1997 committed:
Removed .html file

kohyida1997 committed:
Changed Header files

kohyida1997 committed:
Fixed trailing whitespace

kohyida1997 committed:
Changed Header Files

Merge pull request #222 from AY1920S1-CS2103T-T11-2/yida-5-MoreDocs

sreesubbash committed:
Clean up commented tests

sreesubbash committed:
Clean up funtional code

sreesubbash committed:
PPP is now 10 pages

chrischenhui committed:
Add more documentations

chrischenhui committed:
Add more documentations

sreesubbash committed:
Merge pull request #223 from AY1920S1-CS2103T-T11-2/subbash-29-finalCodeUpdate

Subbash 29 final code update

chrischenhui committed:
Pass checkstyle

chrischenhui committed:
Pass lots of card command tests

chrischenhui committed:
Add some more tests

chrischenhui committed:
Fix checkstyle

chrischenhui committed:
Update gradlew for travis

chrischenhui committed:
Remove some gradlew file

chrischenhui committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into chenhui-3-UGDG

chrischenhui committed:
Fix build

chrischenhui committed:
Merge pull request #224 from AY1920S1-CS2103T-T11-2/chenhui-3-UGDG

Chenhui 3 ugdg

jascxx committed:
Update tests

jascxx committed:
Fix style

jascxx committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into jason-2-ppp

kohyida1997 committed:
PPP, UG and DG edits

kohyida1997 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into yida-5-MoreDocs

kohyida1997 committed:
Cleaning up functional code

kohyida1997 committed:
Merge pull request #226 from AY1920S1-CS2103T-T11-2/yida-5-MoreDocs

Housekeeping of UG and DG

therealdragontho committed:
Applied minor fix to appSettings tampering

therealdragontho committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into paul-8-ppp

chrischenhui committed:
Some minor bug fix, and documentation

chrischenhui committed:
Fix test and pass checkstyle

therealdragontho committed:
Changed timer pictures to reflect the proper decimal points.

chrischenhui committed:
Finalise PPP

chrischenhui committed:
Add new line to fix travis

chrischenhui committed:
Fix spelling

chrischenhui committed:
Merge pull request #228 from AY1920S1-CS2103T-T11-2/chenhui-3-UGDG

Chenhui 3 ugdg

kohyida1997 committed:
PPP Updates

kohyida1997 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into yida-5-MoreDocs

kohyida1997 committed:
PPP Updates

kohyida1997 committed:
Merge pull request #229 from AY1920S1-CS2103T-T11-2/yida-5-MoreDocs

PPP Edits

kohyida1997 committed:
Manual Testing Instructions in DG

kohyida1997 committed:
Merge pull request #230 from AY1920S1-CS2103T-T11-2/yida-5-MoreDocs

Manual Testing Instructions in DG

therealdragontho committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into paul-8-ppp

therealdragontho committed:
Changed Ui diagrams to adapt to new shadowing

dragontho committed:
Merge pull request #227 from AY1920S1-CS2103T-T11-2/paul-8-ppp

Changed more docs

sreesubbash committed:
Minor change, diagram is larger

sreesubbash committed:
Merge pull request #231 from AY1920S1-CS2103T-T11-2/subbash-29-finalCodeUpdate

Minor change, diagram is larger

chrischenhui committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into chenhui-3-UGDG

chrischenhui committed:
Improve consistency in UGDG

kohyida1997 committed:
yida-5-MoreDocs

chrischenhui committed:
Improve consistency in DG

kohyida1997 committed:
Added DG Manual Testing Instructions

DG Manual Testing Instructions

chrischenhui committed:
Update PPP to reflect changes in UGDG

chrischenhui committed:
Merge pull request #233 from AY1920S1-CS2103T-T11-2/chenhui-3-UGDG

Chenhui 3 ugdg

kohyida1997 committed:
Updated Icons in UG

kohyida1997 committed:
Resolved merge conflict

kohyida1997 committed:
Moved .png credits

kohyida1997 committed:
Merge pull request #234 from AY1920S1-CS2103T-T11-2/yida-5-MoreDocs

Fixed icons in UG

kohyida1997 committed:
Updated ReadMe.adoc with .png credits (Doc)

chrischenhui committed:
Update PPP links to DG

chrischenhui committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into chenhui-3-UGDG

chrischenhui committed:
Merge pull request #235 from AY1920S1-CS2103T-T11-2/chenhui-3-UGDG

Chenhui 3 ugdg

chrischenhui committed:
Update PPP's UG link

chrischenhui committed:
Add new line character

chrischenhui committed:
Fix image size

chrischenhui committed:
Final fix to PPP, hopefully

chrischenhui committed:
Remove space

chrischenhui committed:
Merge pull request #236 from AY1920S1-CS2103T-T11-2/chenhui-3-UGDG

Chenhui 3 ugdg

therealdragontho committed:
Made minor grammar adjustments

therealdragontho committed:
Merged conflicts and fixed broken links

dragontho committed:
Merge pull request #237 from AY1920S1-CS2103T-T11-2/paul-8-ppp

Did some housekeeping and fixed some broken links

sreesubbash committed:
Correct Use cases

sreesubbash committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into subbash-29-finalCodeUpdate

sreesubbash committed:
Merge pull request #238 from AY1920S1-CS2103T-T11-2/subbash-29-finalCodeUpdate

Update Use cases

jascxx committed:
Resize diagram

jascxx committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into jason-2-ppp

jascxx committed:
Merge pull request #225 from AY1920S1-CS2103T-T11-2/jason-2-ppp

Update PPP and some tests

chrischenhui committed:
Update DG, PPP language consistency

chrischenhui committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into chenhui-3-UGDG

chrischenhui committed:
Merge pull request #239 from AY1920S1-CS2103T-T11-2/chenhui-3-UGDG

Chenhui 3 ugdg

chrischenhui committed:
Italicise some words

chrischenhui committed:
Merge pull request #240 from AY1920S1-CS2103T-T11-2/chenhui-3-UGDG

Italicise some words

sreesubbash committed:
Small case png

sreesubbash committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into subbash-29-finalCodeUpdate

sreesubbash committed:
PNG to png

Subbash 29 final code update

kohyida1997 committed:
UG formatting for .pdf

kohyida1997 committed:
Merging with master

kohyida1997 committed:
Added page breaks to UG

kohyida1997 committed:
Fixed EOF error

kohyida1997 committed:
Merge pull request #242 from AY1920S1-CS2103T-T11-2/yida-5-MoreDocs

Added Page Breaks for UG

kohyida1997 committed:
Fix PPP Broken Link

kohyida1997 committed:
Final UGDG formatting

chrischenhui committed:
Italicise some words

kohyida1997 committed:
Merge pull request #243 from AY1920S1-CS2103T-T11-2/yida-5-MoreDocs

Final formatting edits for DG

chrischenhui committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-2/main into chenhui-3-UGDG

chrischenhui committed:
Remove ppp pdf

chrischenhui committed:
Merge pull request #244 from AY1920S1-CS2103T-T11-2/chenhui-3-UGDG

Chenhui 3 ugdg

chrischenhui committed:
Add back PPP

chrischenhui committed:
Merge pull request #245 from AY1920S1-CS2103T-T11-2/chenhui-3-UGDG

Add back PPP

chrischenhui committed:
Fix PPP adoc and pdf

chrischenhui committed:
Add new line

chrischenhui committed:
Merge pull request #246 from AY1920S1-CS2103T-T11-2/chenhui-12-pppFix

Fix PPP adoc and pdf
